site/docs/v1/tech/guides/breached-password-detection.adoc

```diff
@@ -0,0 +1,274 @@
+---
+layout: doc
+title: Breached Password Detection
+description: Learn how to create a passwordless experience for your end users.
+---
+:page-liquid:
+
+== What Is Breached Password Detection
+
+Breached password detection consists of the following tasks:
+
+. Finding breached and compromised passwords.
+. Building a system to download, process and store these datasets.
+. Checking passwords to see if they've been compromised.
+. Taking action when a leaked credential is found.
+
```

Comment:
We use the term breached, compromised and then leaked here. Does it make sense to use "compromised" here as well? 

> Taking action when a compromise has been detected

Or something like that?